README.md

```diff
@@ -91,12 +91,14 @@ $ minikube addons disable metrics-server
 
 The following versions are supported and work as we test against these versions in their respective branches. But note that other versions might work!
 
-| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 |
-|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|
-| `release-0.3`         | ✔              | ✔              | ✔              | ✔              | ✗
-| `release-0.4`         | ✗              | ✗              | ✔ (v1.16.5+)   | ✔              | ✗
-| `release-0.5`         | ✗              | ✗              | ✗              | ✗              | ✔
-| `HEAD`                | ✗              | ✗              | ✗              | ✗              | ✔
+| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 | Kubernetes 1.19 |
+|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| `release-0.3`         | ✔               | ✔               | ✔               | ✔               | ✗               | ✗               |
+| `release-0.4`         | ✗               | ✗               | ✔ (v1.16.5+)    | ✔               | ✗               | ✗               |
+| `release-0.5`         | ✗               | ✗               | ✗               | ✗               | ✔               | ✗               |
+| `release-0.5`         | ✗               | ✗               | ✗               | ✗               | ✔               | ✗               |
+| `release-0.6`         | ✗               | ✗               | ✗               | ✗               | ✔               | ✔               |
```

Comment:
What we tend to do is to add the newly supported version to [travis-e2e.sh](https://github.com/coreos/kube-prometheus/blob/master/tests/e2e/travis-e2e.sh) as it was done for [release-0.4](https://github.com/coreos/kube-prometheus/blob/release-0.4/tests/e2e/travis-e2e.sh).

Reply:
Not sure 1.19 only has rc not sure those are available yet on kind? If yes then lets add it.

Reply:
@dgrisonnet unfortunately kind does not yet seem to have 1.19 images, hence I suggest to do this as a follow-up. I'll open an issue here, does that sound good to you?

Reply:
As the kind image for 1.19 has not been released yet, I don't think it is right to add it to the matrix yet as we won't have any mechanism to ensure the compatibility.

Reply:
i am fine with removing that entry, but then it most likely is confusing to users, wdyt @lilic @paulfantom ?

Reply:
we test this in openshift, against 1.19 but yes I understand the concern here. Really don't mind either way. Lets add it when 1.19 is out then?

Reply:
removed the line, left the 1.19 column for future additions.